ui/src/tool-panel/alignment-plan-section-infobox-content.tsx

```diff
@@ -40,6 +42,31 @@ export const AlignmentPlanSectionInfoboxContent: React.FC<
     const { t } = useTranslation();
 
     const delegates = React.useMemo(() => createDelegates(TrackLayoutActions), []);
+    const visiblePlans = useTrackLayoutAppSelector((state) => state.selection.visiblePlans);
+    const _changeTimes = getChangeTimes();
```

Comment:
Jos näitä ei käytetä, niin tämän haun voisi poistaa. Jos käytetään, niin voisi ottaa nimestä alaviivan pois alusta.